### PR TITLE
Command timeout and other settings configurable

### PR DESCRIPTION
**Pull request by mpaladugu-hchb**

### CODE REVIEW

File: SqlReplay.Console/SqlReplay.Console.csproj [hunk 1]
```diff
@@ -7,6 +7,8 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.Configuration.Binder" Version="5.0.0-rc.2.20475.5" />
```
Comment (Balfa) [inline SqlReplay.Console/SqlReplay.Console.csproj:10]: Any special reason we're using these prerelease packages?

Comment (Balfa) [inline SqlReplay.Console/SqlReplay.Console.csproj]: Any special reason we're using these prerelease packages?

**Comment by mpaladugu-hchb** [comment]
We need the package to bind the class to config section in Console app.

Thanks,
Manjula

File: SqlReplay.Console/SqlReplay.Console.csproj (hunk 1 above)
Comment (mpaladugu-hchb) [reply SqlReplay.Console/SqlReplay.Console.csproj]: No special reason, let me fix it.

**Review by Balfa — CHANGES_REQUESTED** [review]
Approved! 🚀

**Pull request merged**